.github/workflows/validate-action-typings.yml

```diff
@@ -0,0 +1,14 @@
+name: Validate action typings
+
+on:
+  push:
+    branches: [master]
```

Comment:
Do you think it should also run on PRs so that we verify before merging?

Reply:
Yes, definitely. Thanks to this, if you change something in the action inputs (e. g. add a new input/output or rename some input), this action will catch any discrepancy between the inputs/outputs and the typings.